### PR TITLE
add vector search operation and update integrations

### PR DESCRIPTION
**Pull request by heibaijian**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/data-service-manage-endpoint.md [hunk 1]
```diff
@@ -56,6 +63,7 @@ In TiDB Cloud Data Service, you can generate one or multiple endpoints automatic
     - Endpoint name: the generated endpoint name is in the `/<name of the selected table>` format, and the request method (such as `GET`, `POST`, and `PUT`) is displayed before the endpoint name. For example, if the selected table name is `sample_table` and the selected operation is `POST (Create)`, the generated endpoint is displayed as `POST /sample_table`.
 
         - If a batch operation is selected, TiDB Cloud Data Service appends `/bulk` to the name of the generated endpoint. For example, if the selected table name is `/sample_table` and the selected operation is `POST (Batch Create)`, the generated endpoint is displayed as `POST /sample_table/bulk`.
+        - If `POST (Vector Similarity Search)` is selected, TiDB Cloud Data Service appends `/vector_search` to the name  of the generated endpoint. For example, if the selected table name is `/sample_table` and the selected operation is `POST (Vector Similarity Search)`, the generated endpoint is displayed as `POST /sample_table/vector_search`.
```
Comment (Oreoxmt) [inline tidb-cloud/data-service-manage-endpoint.md:66]:
```suggestion
        - If `POST (Vector Similarity Search)` is selected, TiDB Cloud Data Service appends `/vector_search` to the name of the generated endpoint. For example, if the selected table name is `/sample_table` and the selected operation is `POST (Vector Similarity Search)`, the generated endpoint is displayed as `POST /sample_table/vector_search`.
```

File: tidb-cloud/data-service-manage-endpoint.md [hunk 2]
```diff
@@ -42,8 +42,15 @@ In TiDB Cloud Data Service, you can generate one or multiple endpoints automatic
 
     2. Select at least one HTTP operation (such as `GET (Retrieve)`, `POST (Create)`, and `PUT (Update)`) for the endpoint to be generated.
 
-        For each operation you selected, TiDB Cloud Data Service will generate a corresponding endpoint. If you have selected a batch operation (such as `POST (Batch Create)`), the generated endpoint lets you operate on multiple rows in a single request.
+        For each operation you select, TiDB Cloud Data Service will generate a corresponding endpoint. If you select a batch operation (such as `POST (Batch Create)`), the generated endpoint lets you operate on multiple rows in a single request.
 
+        If your table contains [vector data types](/tidb-cloud/vector-search-data-types.md), you can click the **Vector Search Operations** toggle to generate a `POST (Vector Similarity Search)` endpoint, which automatically calculates vector distances based on the selected distance function. You can select from the following [vector distance functions](/tidb-cloud/vector-search-functions-and-operators.md):
```
Comment (Oreoxmt) [inline tidb-cloud/data-service-manage-endpoint.md:47]:
```suggestion
        If the table you selected contains [vector data types](/tidb-cloud/vector-search-data-types.md), you can click the **Vector Search Operations** toggle to generate a `POST (Vector Similarity Search)` endpoint, which automatically calculates vector distances based on the selected distance function. You can select from the following [vector distance functions](/tidb-cloud/vector-search-functions-and-operators.md):
```

File: tidb-cloud/data-service-manage-endpoint.md [hunk 3]
```diff
@@ -42,8 +42,15 @@ In TiDB Cloud Data Service, you can generate one or multiple endpoints automatic
 
     2. Select at least one HTTP operation (such as `GET (Retrieve)`, `POST (Create)`, and `PUT (Update)`) for the endpoint to be generated.
 
-        For each operation you selected, TiDB Cloud Data Service will generate a corresponding endpoint. If you have selected a batch operation (such as `POST (Batch Create)`), the generated endpoint lets you operate on multiple rows in a single request.
+        For each operation you select, TiDB Cloud Data Service will generate a corresponding endpoint. If you select a batch operation (such as `POST (Batch Create)`), the generated endpoint lets you operate on multiple rows in a single request.
 
+        If the table you selected contains [vector data types](/tidb-cloud/vector-search-data-types.md), you can click the **Vector Search Operations** toggle to generate a `POST (Vector Similarity Search)` endpoint, which automatically calculates vector distances based on the selected distance function. You can select from the following [vector distance functions](/tidb-cloud/vector-search-functions-and-operators.md):
```
Comment (qiancai) [inline tidb-cloud/data-service-manage-endpoint.md:47]:
```suggestion
        If the table you selected contains [vector data types](/tidb-cloud/vector-search-data-types.md), you can enable the **Vector Search Operations** option and select a vector distance function to generate a vector search endpoint that automatically calculates vector distances based on your selected distance function. The supported [vector distance functions](/tidb-cloud/vector-search-functions-and-operators.md) include the following:
```

File: tidb-cloud/data-service-integrations.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+5. Use the copied API Specification URL and the encoded API key in your GPT configuration.
+
+## Integrate your Data App with Dify
+
+You can integrate your Data App with [Dify](https://docs.dify.ai/guides/tools) to enhance your applications with intelligent capabilities.
+
+Integrating your Data App with Dify enhances your applications with intelligent capabilities, including vector distance calculations. This integration enables advanced similarity searches and vector analysis.
+
+To integrate your Data App with Dify, follow the same steps as for [GPTs integration](#integrate-your-data-app-with-gpts).
```
Comment (qiancai) [inline tidb-cloud/data-service-integrations.md:42]:
```suggestion
To integrate your Data App with Dify, follow the same steps as for [GPTs integration](#integrate-your-data-app-with-gpts). The only difference is that on the **Integrations** tab, you need to click **Get Configuration** in the **Integrate with Dify** area.
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**